.github/workflows/java-common-release-build-pull-request.yml

```diff
@@ -0,0 +1,201 @@
+name: Release Pull Request
```

Comment:
is this a duplicate of .github/workflows/shared-release-pull-request-java.yml ?

Reply:
It isn't. Service and library CI/CDs are quite different.